# generated by patch-package 6.5.0 on 2022-03-31 20:04:11
#
# command:
#   npx patch-package @emotion/cache --exclude nothing
#
# declared package:
#   @emotion/cache: 11.7.1
#
diff --git a/node_modules/@emotion/cache/package.json b/node_modules/@emotion/cache/package.json
index 437f0bf..c74881a 100644
--- a/node_modules/@emotion/cache/package.json
+++ b/node_modules/@emotion/cache/package.json
@@ -4,10 +4,6 @@
   "description": "emotion's cache",
   "main": "dist/emotion-cache.cjs.js",
   "module": "dist/emotion-cache.esm.js",
-  "browser": {
-    "./dist/emotion-cache.cjs.js": "./dist/emotion-cache.browser.cjs.js",
-    "./dist/emotion-cache.esm.js": "./dist/emotion-cache.browser.esm.js"
-  },
   "types": "types/index.d.ts",
   "license": "MIT",
   "repository": "https://github.com/emotion-js/emotion/tree/main/packages/cache",
